pages/wysiwyg.html

```diff
@@ -100,10 +100,8 @@ <h1>Builder</h1>
       </section>
       <section class="query-builder--preview">
         <form action="#">
-          <div class="field field--textarea">
-            <label for="query">Query</label>
-            <textarea name="query" id="" cols="30" rows="1" placeholder="Type a search, or use the tools above to get started."></textarea>            
-          </div>
+          <label for="query">Query</label>
+          <textarea name="query" id="" cols="30" rows="1" placeholder="Type a search, or use the tools above to get started."></textarea>
```

Comment:
I'd revert this. The label/input pairs are supposed to have a wrapping element in a form.